completions/ip

```diff
@@ -289,7 +302,20 @@ _comp_cmd_ip()
                     # TODO
                     ;;
                 show | flush)
-                    # TODO
+                    case "$prev" in
+                        nud)
+                            _comp_cmd_ip__neigh_states "$1"
+                            ;;
+                        dev)
+                            _comp_compgen_available_interfaces "$1"
```

Comment:
This passes the command being completed to `_comp_compgen_available_interfaces`, but that's not the correct arg it takes. It would be kind of nice if it invoked the same `ip` we are invoking here, but as it stands, that's not available.

```suggestion
                            _comp_compgen_available_interfaces
```

Reply:
Done